### PR TITLE
Roll out 33.20210315.1.0, 33.20210314.2.0, 33.20210301.3.1

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

**Comment by bgilbert** [comment]
```
testing
    version: 33.20210314.2.0
    start: 2021-03-17 16:00:00+00:00 UTC (in 0:30:02)
           2021-03-17 12:00:00-04:00 Raleigh/New York/Toronto
           2021-03-17 17:00:00+01:00 Berlin/France/Poland
    duration: 2880m (48.0h)
stable
    version: 33.20210301.3.1
    start: 2021-03-17 16:00:00+00:00 UTC (in 0:30:02)
           2021-03-17 12:00:00-04:00 Raleigh/New York/Toronto
           2021-03-17 17:00:00+01:00 Berlin/France/Poland
    duration: 2880m (48.0h)
next
    version: 33.20210315.1.0
    start: 2021-03-17 16:00:00+00:00 UTC (in 0:30:02)
           2021-03-17 12:00:00-04:00 Raleigh/New York/Toronto
           2021-03-17 17:00:00+01:00 Berlin/France/Poland
    duration: 2880m (48.0h)
```

File: streams/next.json [hunk 1]
```diff
@@ -1,192 +1,182 @@
...
                 "aliyun": {
-                    "release": "33.20210302.1.0",
+                    "release": "33.20210315.1.0",
                     "formats": {
                         "qcow2.xz": {
                             "disk": {
-                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/33.20210302.1.0/x86_64/fedora-coreos-33.20210302.1.0-aliyun.x86_64.qcow2.xz",
-                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/33.20210302.1.0/x86_64/fedora-coreos-33.20210302.1.0-aliyun.x86_64.qcow2.xz.sig",
-                                "sha256": "efe83c1876665fdf807ef2bc0c8d5cf2ac833c95601027e2d5e6872ba13105b2",
-                                "uncompressed-sha256": "7f1aedc4f00a72b40ca5d911fe42c9015e1a32d32b7c8302fe7a1be6b058585c"
```
Comment (jlebon) [inline streams/next.json:17]: We're losing the `uncompressed-sha256` fields. Out-of-date `fedora-coreos-stream-generator`?

Comment (bgilbert) [reply streams/next.json:17]: Looks like the generator was never revendored to add the field, but it somehow slipped into the last release anyway.  https://github.com/coreos/fedora-coreos-stream-generator/pull/22

**Comment by bgilbert** [comment]
Updated:

```
testing
    version: 33.20210314.2.0
    start: 2021-03-17 17:00:00+00:00 UTC (in 0:43:49)
           2021-03-17 13:00:00-04:00 Raleigh/New York/Toronto
           2021-03-17 18:00:00+01:00 Berlin/France/Poland
    duration: 2880m (48.0h)
stable
    version: 33.20210301.3.1
    start: 2021-03-17 17:00:00+00:00 UTC (in 0:43:48)
           2021-03-17 13:00:00-04:00 Raleigh/New York/Toronto
           2021-03-17 18:00:00+01:00 Berlin/France/Poland
    duration: 2880m (48.0h)
next
    version: 33.20210315.1.0
    start: 2021-03-17 17:00:00+00:00 UTC (in 0:43:48)
           2021-03-17 13:00:00-04:00 Raleigh/New York/Toronto
           2021-03-17 18:00:00+01:00 Berlin/France/Poland
    duration: 2880m (48.0h)
```

**Review by jlebon — APPROVED** [review]

**Pull request merged**